src/coreclr/src/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -6,14 +6,13 @@
     <GenerateResxSourceOmitGetResourceString>true</GenerateResxSourceOmitGetResourceString>
     <GenerateNeutralResourcesLanguageAttribute>false</GenerateNeutralResourcesLanguageAttribute>
     <EnsureRuntimePackageDependencies>false</EnsureRuntimePackageDependencies>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <TargetFramework Condition="'$(BuildingInsideVisualStudio)' == 'true'">netcoreapp3.0</TargetFramework>
```

Comment:
Why can't we build for nca5.0 in visual studio?

Reply:
Unsure, see https://github.com/dotnet/runtime/pull/584 for discussion.